jenkins/gradle/gradle-check.jenkinsfile

```diff
@@ -150,28 +152,38 @@ pipeline {
                         def invokedBy
                         def pullRequest
                         def pullRequestTitle
+                        def gitReference
+                        def pullRequestOwner
                         switch (true) {
                             case env.BUILD_CAUSE.contains('Started by user'):
                                 invokedBy = 'User'
-                                pullRequest = "${GIT_REFERENCE}"
-                                pullRequestTitle = "Null"
+                                pullRequest = "null"
+                                pullRequestTitle = "null"
+                                gitReference = "${GIT_REFERENCE}"
+                                pullRequestOwner = "null"
                                 break
                             case env.BUILD_CAUSE.contains('Started by timer'):
                                 invokedBy = 'Timer'
-                                pullRequest = "${GIT_REFERENCE}"
-                                pullRequestTitle = "Null"
+                                pullRequest = "null"
+                                pullRequestTitle = "null"
+                                gitReference = "${GIT_REFERENCE}"
+                                pullRequestOwner = "null"
                                 break
-                            case "${pr_number}" == "Null":
+                            case "${post_merge_action}" == "true":
                                 invokedBy = 'Post Merge Action'
-                                pullRequest = "${GIT_REFERENCE}"
+                                pullRequest = "${pr_number}"
                                 pullRequestTitle = "${pr_title}"
+                                gitReference = "${pr_from_sha}"
+                                pullRequestOwner = "${pr_owner}"
                                 break
                             default:
                                 invokedBy = 'Pull Request'
                                 pullRequest = "${pr_number}"
                                 pullRequestTitle = "${pr_title}"
+                                gitReference = "${pr_from_sha}"
+                                pullRequestOwner = "${pr_owner}"
                         }
-                        publishGradleCheckTestResults(prNumber: "${pullRequest}" , prDescription: "${pullRequestTitle}", invokeType: "${invokedBy}")
+                        publishGradleCheckTestResults(prNumber: "${pullRequest}" , prTitle: "${pullRequestTitle}", prOwner: "${pullRequestOwner}", invokeType: "${invokedBy}", gitReference: "${gitReference}")
```

Comment:
nit: Will gitReference be confusing or should it be commit_id? Just curious. Look good overall.

Reply:
Sayali initially I have added `commit_id`, but during gradle check job execution when scheduled with timer it takes the branch as `main`, `2.x`, then this would deviate with the term `commit_id`, hence added as gitReference as its generic.